scripts/installdeps.sh

```diff
@@ -14,28 +14,40 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-if [ "$(uname)" == "Darwin" ]; then
-    PACKPLATFORM="macos"
-    KUBECTLPLATFORM="darwin"
-    OPERATORSDKPLATFORM="apple-darwin"
-    echo "Once this installation finishes, please do install docker from https://docs.docker.com/docker-for-mac/install/"
-elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
-    PACKPLATFORM="linux"
-    KUBECTLPLATFORM="linux"
-    OPERATORSDKPLATFORM="linux-gnu"
-    if ! grep -q container_t "/proc/1/attr/current"; then
+# initOS discovers the operating system for this system.
+initOS() {
+    OS="$(uname | tr '[:upper:]' '[:lower:]')"
+
+    case "$OS" in
+    # Minimalist GNU for Windows
+    mingw*) OS='windows' ;;
+    esac
+}
+
+initOS
+
```

Comment:
Can we add a line to get acknowledgement from user that these binaries will be installed? Also, warn them that if it is already installed, it might be updated?

Reply:
`Can we add a line to get acknowledgement from user that these binaries will be installed?` Done

Not sure about warning them about updates. We are not updating anything.